04-base-r.Rmd

```diff
@@ -1,4 +1,109 @@
-# Manipulating Dataframes
+# Base R
+
+R comes with a number of functions that allow you to maniuplate, summarise and analyse data. Collectively these are known as **Base R**.
+
+This chapter summarises some of the core functions for manipulating data in base R.
+
+## Loading in data
+
+The most common process to load in data is to load in a CSV file.
+
+```{r sec4_1,echo=FALSE}
+#Load in data
+swfc_16_init <- read.csv("data/SWFC_2016_Machine_Readable.csv")
```

Comment:
At this point of loading data, can we add to reading in .csv's - `stringsasfactors = FALSE` ? Will help a lot with the consistency of the dplyr chapter 6!

Reply:
And same for the 2nd loading .csv - will just really help when I'm describing variable types

Reply:
I don’t think we should do that - factors are useful for stuff like regression. Can you make clear in the next chapter that when you are not running statistical models that factors tend to be of less use instead

Reply:
Yeah that's fine. It was mainly for summary statistics and then casting variables etc.